front/src/common/IntervalsEditor/IntervalsEditorCommonForm.tsx

```diff
@@ -15,6 +15,7 @@ type IntervalsEditorFormProps = {
   setData: (newData: IntervalItem[]) => void;
   setSelectedIntervalIndex: (selectedIntervalIndex: number) => void;
   totalLength: number;
+  defaultValue?: string;
```

Comment:
Why do you authorize undefined ? I think it's better for now to make it required. Also, you can make it either a string or a number
```suggestion
  defaultValue: string | number;
```

Reply:
fixed ! ty